client/src/utils/curriculum-helpers.js

```diff
@@ -1,7 +1,7 @@
 import { parse } from '@babel/parser';
 import generate from '@babel/generator';
 
-const removeHtmlComments = str => str.replace(/<!--(.|\s)*?-->/g, '');
+const removeHtmlComments = str => str.replace(/<!--(.|\r|\n)*?-->/g, '');
```

Comment:
```suggestion
const removeHtmlComments = str => str.replace(/<!--.*?-->/gs, '');
```
Could we use this instead? We don't care about the capturing groups, after all.

Reply:
Yes, that also works. 
And it is a simpler solution :+1: